content/docs/user-guide/networking/multiplayer/MultiplayerVersionMismatch.md

```diff
@@ -0,0 +1,26 @@
+# Multiplayer Version Mismatch
+
+In order to keep the multiplayer simulation in sync, it's important that all connected multiplayer endpoints are running the same multiplayer version.
+
+For example, suppose a server is running a build an old multiplayer player component. When a client's player tries to sends new network input to the server, the server won't know how to handle the input. Servers and clients must be running the same version of all the multiplayer components (components which communicate to each other over the network); any differences can lead to unexpected behavior. The multiplayer versioning feature will warn users upon finding a mismatch.
+
+## How to enable the Multiplayer Version Mismatch feature:
+Multiplayer version mismatch detection is enabled automatically inside the [Multiplayer Gem](/docs/user-guide/gems/reference/multiplayer/). If two multiplayer endpoints connect with different versions, a Multiplayer::VersionMismatchEvent [AZ::Event](/docs/user-guide/programming/az-event/) will be triggered, and debug logs containing information about which auto-components are mismatched will be printed to the console.
+
+## Relevant Console Variables (CVars)
+**sv_versionMismatch_autoDisconnect**  *true | false* Determines if a mismatched connection with automatically terminate. It's recommended to keep this true; even minor differences between the version of a multiplayer component can cause unexpected behavior.
+
+**sv_versionMismatch_sendManifestToClient**  *true | false* Determines if the server will send all its individual multiplayer component version information to the client when there's a mismatch. Upon receiving the information, the client will print which components are different to the console. Provided for debugging during development, and is suggested to mark false for release builds in order than clients to not have access to any multiplayer component information that should be kept private (component names and version hashes).
```

Comment:
```suggestion
**sv_versionMismatch_sendManifestToClient**  *true | false* Determines if the server will send all its individual multiplayer component version information to the client when there's a mismatch. Upon receiving the information, the client will print which components are different to the console. Its recommended to set to false for release builds. This is to prevent clients having knowledge to any multiplayer component information that should be kept private (component names and version hashes).
```